threejs/three-FirstPersonControls.d.ts

```diff
@@ -11,21 +11,21 @@ declare namespace THREE {
         object: THREE.Object3D;
         target: THREE.Vector3;
         domElement: HTMLCanvasElement;
+        enabled: boolean;
         movementSpeed: number;
         lookSpeed: number;
         noFly: boolean;
         lookVertical: boolean;
         autoForward: boolean;
         activeLook: boolean;
         heightSpeed: boolean;
-        heightCoef: boolean;
-        heightMin: boolean;
+        heightCoef: number;
+        heightMin: number;
+        heightMax: number;
         constrainVertical: boolean;
         verticalMin: number;
         verticalMax: number;
         autoSpeedFactor: number;
-        mouseX: number;
```

Comment:
i wold leave these unless we have a clear statement from the package author that they are private to avoid breaking other users.